### PR TITLE
Fixed #27250 -- Removed 'for ="..."' from CheckboxSelectMultiple's <label>.

### PR DESCRIPTION
**Pull request by martinarroyo**
Removes number appending on MultipleChoiceField

### CODE REVIEW

**Comment by martinarroyo** [comment]
@apollo13 As discussed, here is our PR ;)

**Comment by charettes** [comment]
What about removing the `for=` completely instead?

**Comment by timgraham** [comment]
Also since you're only modifying the widget rather than the field, `forms_tests/widget_tests` might be a better place for the test.

**Comment by timgraham** [comment]
buildbot, add to whitelist.

File: django/forms/widgets.py [hunk 1]
```diff
@@ -822,6 +822,12 @@ def value_omitted_from_data(self, data, files, name):
         # never known if the value is actually omitted.
         return False
 
+    # Check ticket 27250
```
Comment (timgraham) [inline django/forms/widgets.py]: Please use descriptive comments rather than a reference to the ticket.

**Comment by martinarroyo** [comment]
@charettes We tried that initially, as noted in the issue tracker, but then some tests started to fail, so we compromised on this current solution.

**Comment by timgraham** [comment]
Could you give details of the test failures? Maybe they just need to be adapted for the new behavior.

**Comment by charettes** [comment]
> Maybe they just need to be adapted for the new behavior.

That's also what I suspect.

**Comment by timgraham** [comment]
I added a commit that hopefully addresses the issue.

**Pull request merged**